web/src/stream_types.ts

```diff
@@ -10,6 +10,9 @@ export const enum StreamPostPolicy {
 }
 
 export type stream_permission_group_setting_type = "can_remove_subscribers_group";
```

Comment:
This type can be created using the zod schema as mentioned above. And also we should use CamelCase for the variable without having "type" in it like we do for other types in `state_data.ts`.